package-lock.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "roaster",
-  "version": "1.1.4",
+  "version": "1.5.0",
```

Comment:
Note to self: I am wondering why the package version number was not reflected in the package-lock.json before.
Nevertheless, this file should not be part of this commit. Are you able to remove it?